AGENTS.md

```diff
@@ -240,8 +289,8 @@ All visible content must be fully rendered server-side by the build system. User
 
 **Implementation Details**:
 
-- Jekyll/Ruby: See [.github/agents/fullstack.md](.github/agents/fullstack.md) for server-side patterns
-- Ruby Plugins: See [_plugins/AGENTS.md](_plugins/AGENTS.md) for plugin-specific handling
+- Server-side: See [.github/agents/fullstack.md](.github/agents/fullstack.md) for server-side patterns
+- Build plugins: See [_plugins/AGENTS.md](_plugins/AGENTS.md) for plugin-specific handling
```

Comment:
Here and in other places: In the current context a plugin is specifically a Jekyll Plugin, please call it that instead of Build plugins. Also check the other changes in this file for this. Also, in the line above: the fullstack file has instructions for complete fullstack development, so also client-side.

So Jekyll specific stuff that has no proper alternative in a C# web-app (like plugins as they exist for Jekyll) should just be called Jekyll. That also makes it clear where things still need changing when moving away from Jekyll